src/main/java/com/joss/conductor/mobile/Config.java

```diff
@@ -31,4 +31,7 @@
     String avd() default "";
     String appActivity() default "";
     String appWaitActivity() default "";
+    boolean fullReset() default false;
+    boolean noReset() default false;
```

Comment:
This actually creates an issue with how default booleans work. For example, adding a `@Config` to a class and only overriding `Platform` and not `fullReset` (or any other boolean field) will override the default value to `false`, even though it's set as `true` in the `default.properties`.

Reply:
The solution might be to use something other than `boolean`, maybe Enums or String?